### PR TITLE
Class6/hamidreza fakharzadeh

### PR DESCRIPTION
**Pull request by Hamidreza-Fakharzadeh**
Hi

### CODE REVIEW

**Comment by Hamidreza-Fakharzadeh** [comment]
hi. it is my first time i make pull request

**Comment by Hamidreza-Fakharzadeh** [comment]
hi

**Comment by Hamidreza-Fakharzadeh** [comment]
i am trying to make a pull request

File: index.html [hunk 1]
```diff
@@ -12,6 +12,28 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header>
```
Comment (AlfioEmanueleFresta) [inline index.html]: Nice navigation structure!

Comment (Hamidreza-Fakharzadeh) [reply index.html]: Thank you

File: index.html [hunk 2]
```diff
@@ -12,6 +12,28 @@
...
+                <li> <a>Store</a> </li>
+                <li> <a>Home</a> </li>
+                <li> <a>Blog</a> </li>
+                <li> <a>Help</a> </li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="hero-sec">
+            <img class="hero-img" src="img/first-background.jpg" alt="myPic" >
```
Comment (AlfioEmanueleFresta) [inline index.html]: Make sure you include a meaningful description of your image in the `alt` attribute. Users of assistive technology such as screen readers rely on these descriptions to be able to *see* your images!

File: index.html [hunk 3]
```diff
@@ -12,6 +12,28 @@
...
+            <article>Article</article>
```
Comment (AlfioEmanueleFresta) [inline index.html]: This is a good start! I would encourage you to keep working on this until it looks similar to the example.

Comment (Hamidreza-Fakharzadeh) [reply index.html]: Thank you. I will try to improve it. Now i am working on "Responsive cake web". I think it has many common things.

**Pull request closed without merging**